docs/data/material/components/steppers/steppers.md

```diff
@@ -22,6 +22,29 @@ Steppers may display a transient feedback message after a step is saved.
 This component is no longer documented in the [Material Design guidelines](https://m2.material.io/), but Material UI will continue to support it.
 :::
 
+## Introduction
+
+The Stepper component displays progress through a sequence of logical and numbered steps.
+It supports horizontal and vertical orientation for desktop and mobile viewports.
+
+Steppers are implemented using a collection of related components:
+
+- Stepper: the container for the steps. Renders as a `<ol>` by default.
+- Step: an individual step in the sequence. Renders as a `<li>` by default.
```

Comment:
`ol` and `li` are not the defaults. It renders a `div` by default, which we shouldn't mention:
```suggestion
- Stepper: the container for the steps.
- Step: an individual step in the sequence.
```